### ISSUE TITLE
[Bug]: when using cscope, no result is shown in console

### ISSUE THREAD

**Issue by lyfadvance** [eranif/codelite#3181]
### What happened?

When I run the cscope plug-in, the ide does not display the results.
 However, when I open the debug log, the log contains the running results of the cscope. 
This seems to be a communication issue with the program interface, but I have upgraded libssh to 0.10.4

### Version

16.0.2

### Operating system

Linux

**Comment by dghart** [comment]
Just to confirm that I see the same thing in 17.0.0 on ubuntu kinetic. However it works normally in 16.7.0 on debian 11.

**Comment by lyfadvance** [comment]
Yes, the system I am using is ubuntu 18.04. In addition, the libssh version of ubuntu 18.04 is 0.8. When compiling codelite from source code, an error was reported: 'SSH_ OPTIONS_ PUBLICKEY_ ACCEPTED_ TYPES' undeclared。 After upgrading libssh to 1.0, this issue will not occur.

**Comment by dghart** [comment]
I've now had time to investigate this further. I've tested in various distro/versions (in VirtualBox) and, initially, all failed. That includes debian 11, even though my 'real' debian works fine.

I then tried (installing cscope! and) testing the various options: clicking the Update button, selecting the _Update Db if stale_ option,  and the _Create reverted index_ option. Sometimes one of those, sometimes another, sometimes a combination, seemed to work. In the ubuntu 18.04 guest, FWIW it was the Update one that worked. Please experiment and report back what you find.

I'm not suggesting that this is an ideal situation, but it does imply that it's not a broken feature.
